metadata-ingestion/src/datahub/cli/cli_utils.py

```diff
@@ -63,8 +76,50 @@ def get_session_and_host():
 
             gms_host = gms_config.server
             gms_token = gms_config.token
+            return gms_host, gms_token
         except yaml.YAMLError as exc:
             click.secho(f"{DATAHUB_CONFIG_PATH} malformatted, error: {exc}", bold=True)
+    return None, None
+
+
+def get_details_from_env() -> Tuple[Optional[str], Optional[str]]:
+    gms_host: Optional[str] = None
+    try:
+        gms_host = os.environ[ENV_METADATA_HOST]
```

Comment:
you can use `os.environ.get(ENV_METADATA_HOST, None)` to simplify this code, you won't need to handle KeyError.

Reply:
good point